KubeArmor/main.go

```diff
@@ -42,12 +42,13 @@ func main() {
 	// options (boolean)
 	enableHostPolicyPtr := flag.Bool("enableHostPolicy", false, "enabling host policies")
 	enableEnforcerPerPodPtr := flag.Bool("enableEnforcerPerPod", false, "enabling the enforcer per pod")
+	enableKvmAgent := flag.Bool("enableKvmAgent", false, "Flag to enable/disable KVM Agent on runtime")
```

Comment:
let's use pothole case for command-line arguments?
For e.g.
`enable_kvma`
`enable_enforcer_per_pod`
`enable_host_policy`

Reply:
Need to change the existing enable flags as well ? Pothole looks great and readable. Can finalise as per comments from others as well.

Reply:
enable_enforcer_per_pod was removed while enableKubeArmoPolicy and enableKubeArmorHostPolicy were added.
let's update these on the latest code.